src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -319,8 +332,16 @@ public function initialized($id)
         if (isset($this->aliases[$id])) {
             $id = $this->aliases[$id];
         }
+        if (isset($this->services[$id])) {
+            return true;
+        }
+        if (array_key_exists($id, $this->services)) {
+            @trigger_error(sprintf('Checking if a unset "%s" service was initialized is deprecated since Symfony 3.2 and won\'t be supported anymore in Symfony 4.0.', $id), E_USER_DEPRECATED);
```

Comment:
Got your point. Do you propose to leave it as is, or should it be updated to just
`return isset($this->services[$id]);` so you can avoid a possible deprecation errors.